docs/conf.py

```diff
@@ -25,46 +25,46 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.autodoc',
-    'sphinx.ext.intersphinx',
-    'sphinx.ext.todo',
-    'sphinx.ext.doctest',
+    "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinx.ext.doctest",
 ]
```

Comment:
Please undo changing " to '. This only bloats the PR and is not about adding documentation.

Reply:
I will remove the conf.py from the pull-request later.